libs/utils/energy.py

```diff
@@ -320,13 +320,13 @@ class ACME(EnergyMeter):
     BayLibre's ACME board based EnergyMeter
     """
 
-    def __init__(self, target, conf, res_dir):
+    def __init__(self, target, acme_conf, res_dir):
```

Comment:
Do we really need this renaming? That's a conf param for the ACME class, it's quite non ambiguous to me.